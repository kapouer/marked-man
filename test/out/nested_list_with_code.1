A(1)                                                                      A(1)

NNAAMMEE
       aa

       Tight list:

              • ttooggggllee__ssttaattuuss

                • Toggle the display of the status bar1

                • Toggle the display of the status bar2

              • ssppaawwnn  <<eexxeeccuuttaabbllee>>  <<aaddddiittiioonnaall aarrggss>> TODO explain path-alike
                expansion

       Loose list:

              • ttooggggllee__ssttaattuuss

                • Toggle the display of the status bar1

                • Toggle the display of the status bar2

              • ssppaawwnn <<eexxeeccuuttaabbllee>> <<aaddddiittiioonnaall aarrggss>> TODO  explain  path-alike
                expansion

                                 January 1979                             A(1)
