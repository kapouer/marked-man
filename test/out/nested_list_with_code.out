[4mA[24m(1)                        General Commands Manual                       [4mA[24m(1)

[1mNAME[0m
       [1ma[0m

       Tight list:

        ‚Ä¢ [1mtoggle_status[0m

         ‚Ä¢ Toggle the display of the status bar1

         ‚Ä¢ Toggle the display of the status bar2

        ‚Ä¢ [1mspawn  <executable> <additional args> [22mTODO explain path-alike expan‚Äê
          sion

       Loose list:

        ‚Ä¢ [1mtoggle_status[0m

         ‚Ä¢ Toggle the display of the status bar1

         ‚Ä¢ Toggle the display of the status bar2

        ‚Ä¢ [1mspawn <executable> <additional args> [22mTODO explain path-alike  expan‚Äê
          sion

                                 January 1979                             [4mA[24m(1)
